main.go

```diff
@@ -215,7 +215,7 @@ func buildURI(uri string, user string, password string) string {
 		// add user and pass to the uri
 		uri = fmt.Sprintf("%s:%s@%s", user, password, uri)
 	}
-	if !strings.HasPrefix(uri, "mongodb") {
+	if !strings.HasPrefix(uri, "mongodb://") {
```

Comment:
```suggestion
	if !strings.HasPrefix(uri, "mongodb://") &&  !strings.HasPrefix(uri, "mongodb+srv://") {
```

Reply:
could you add to test cases origin which contains mongodb+srv prefix?

Reply:
Hello @BupycHuk, I pushed a new commit that addresses considerations for URIs with the srv prefix, thank you for the review